plugins/core/jetbrains-community/resources/telemetryOverride.json

```diff
@@ -308,6 +308,15 @@
                 "reloaded"
             ],
             "description": "Toolkit run state."
+        },
+        {
+            "name": "authType",
```

Comment:
why not in common?

Reply:
We have diverged from the common implementation of this metric in JB. There is a backlog item to address this divergence